hack/run_test

```diff
@@ -0,0 +1,424 @@
+#!/bin/bash
+#
+# Test the PostgreSQL image.
+#
+# IMAGE_NAME specifies the name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+set -exo nounset
+shopt -s nullglob
+
+test -n "${IMAGE_NAME-}" || false 'make sure $IMAGE_NAME is defined'
+test -n "${VERSION-}" || false 'make sure $VERSION is defined'
```

Comment:
Turns out that the $VERSION is probably not properly exported for 'make test' now.